### DISCUSSION TITLE
Apple Silicon

### DISCUSSION THREAD

**Discussion by macadamia** [openalea/openalea#259] [Q&A]
Are there any specific instructions for getting Openalea up and running on an Mac with the M1 chip. I have the basics, but I’m having trouble with plantgl.

When I run
from openalea.plantgl.all import *

I get errors like
dlopen(/Users/neilwhite/miniconda3/envs/openalea/lib/python3.10/site-packages/OpenAlea.PlantGL-3.20.1-py3.10.egg/openalea/plantgl/scenegraph/_pglsg.so, 0x0002): Library not loaded: @rpath/libjpeg.8.dylib

Any thoughts?

**Comment by pradal** [comment]
@fredboudon : any insights?

**Comment by pradal** [comment]
This has been solved with the last release (today).
